erpnext/crm/doctype/lead/lead_list.js

```diff
@@ -77,3 +86,10 @@ frappe.listview_settings["Lead"] = {
 		}
 	},
 };
+
+function maskPhoneNumber(phone) {
+	const visibleDigits = 5;
```

Comment:
issue: this is still a magic number, so if the issue above is resolved, this issue will be resolved automatically since they're referencing to the same thing